apiv2/internal/server/host.go

```diff
@@ -28,28 +28,32 @@ import (
 	"github.com/open-edge-platform/infra-core/inventory/v2/pkg/validator"
 )
 
-// handle AMT and Power related fields in the *HostRequest and *RegisterHostRequest
-
 // OpenAPIHostToProto maps OpenAPI fields name to Proto fields name.
 // The key is derived from the json property respectively of the
 // structs HostTemplate and HostBmManagementInfo defined in
 // edge-infra-manager-openapi-types.gen.go.
 // Here we should have only fields that are writable from the API.
 var OpenAPIHostToProto = map[string]string{
-	computev1.HostResourceFieldName:    inv_computev1.HostResourceFieldName,
-	computev1.HostResourceFieldSiteId:  inv_computev1.HostResourceEdgeSite,
-	computev1.HostResourceEdgeMetadata: inv_computev1.HostResourceFieldMetadata,
+	computev1.HostResourceFieldName:               inv_computev1.HostResourceFieldName,
+	computev1.HostResourceFieldSiteId:             inv_computev1.HostResourceEdgeSite,
+	computev1.HostResourceEdgeMetadata:            inv_computev1.HostResourceFieldMetadata,
+	computev1.HostResourceFieldPowerCommandPolicy: inv_computev1.HostResourceFieldPowerCommandPolicy,
```

Comment:
are we sure of this? IIRC we should have fields that are writable and when there is a mismatch oapi/proto. @raphaelvrosa  is still the case for apiv2?

Reply:
It should add the fields that are being set when updating a host, i.e.,
```

    DesiredPowerState:  inv_computev1.PowerState(host.GetDesiredPowerState()),
    DesiredAmtState:    inv_computev1.AmtState(host.GetDesiredAmtState()),
    PowerCommandPolicy: inv_computev1.PowerCommandPolicy(host.GetPowerCommandPolicy()),
```